tests/tests_unit.yml

```diff
@@ -1,5 +1,17 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
+- hosts: all
+  name: Setup for test running
+  tasks:
+    # FIXME: change with_items to loop when test-harness is updated
```

Comment:
do you have a specific updatre to test-harness in mind?

Reply:
It needs to be deployed with a newer Ansible version, e.g. using Fedora 28.